src/analysis/retail/paladin/holy/modules/dragonflight/tier/T30TierSet.tsx

```diff
@@ -0,0 +1,136 @@
+import SPELLS from 'common/SPELLS';
+import { TIERS } from 'game/TIERS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateEffectiveHealing, calculateOverhealing } from 'parser/core/EventCalculateLib';
+import Events, { HealEvent, ResourceChangeEvent } from 'parser/core/Events';
+import BoringValueText from 'parser/ui/BoringValueText';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { TALENTS_PALADIN } from 'common/TALENTS';
+import { formatNumber } from 'common/format';
+import HIT_TYPES from 'game/HIT_TYPES';
+import { getLightsHammerHeals } from '../../../normalizers/CastLinkNormalizer';
+import { SpellLink } from 'interface';
+
+const CRIT_HEAL_AMOUNT = 1.65;
+const TWO_PIECE_HEAL_INC = CRIT_HEAL_AMOUNT / 1.5; // 210% heal instead of 150% on crit
```

Comment:
crit heals are 200% in pve.
and if i understand these numbers correctly, the comment includes the most recent buff to 60% extra crit healing but `CRIT_HEAL_AMOUNT` does not?